locations.js

```diff
@@ -621,10 +621,18 @@ class TrashAppInfo extends LocationAppInfo {
     }
 
     _onTrashChange() {
+        if (this._schedUpdateId)
+            GLib.source_remove(this._schedUpdateId);
```

Comment:
Please unset this also on `destroy()`.

We're already cancelling the cancellable there, so indirectly we should avoid to do thing anyways, but still better to trigger the callback.

Reply:
Done.